tests/integration/validate_test.go

```diff
@@ -282,16 +282,34 @@ func Test_Validate_PartialLookupTags(t *testing.T) {
 		name           string
 		stateFile      string
 		additionalArgs []string
+		mode           bool
 		errorExpected  bool
+		errorString    string
 	}{
 		{
 			name:           "validate partials",
 			stateFile:      "testdata/validate/001-partials/partials.yaml",
 			additionalArgs: []string{"--online-entities-list=Partials"},
+			mode:           ONLINE,
 		},
 		{
 			name:      "validate default_lookup_tags with partials",
 			stateFile: "testdata/validate/001-partials/partial-lookup-tags.yaml",
+			mode:      ONLINE,
+		},
+		{
+			name:           "validate partials",
+			stateFile:      "testdata/validate/001-partials/partials.yaml",
+			additionalArgs: []string{"--online-entities-list=Partials"},
```

Comment:
Is the arg relevant for offline validation - since we are anyway printing an error message